docs/counter/Snippets.re

```diff
@@ -0,0 +1,65 @@
+// #region app-v1
+module App = {
+  [@react.component]
+  let make = () => <div> {React.string("welcome to my app")} </div>;
+};
+// #endregion app-v1
+
+// #region use-app-component
+let node = ReactDOM.querySelector("#root");
+switch (node) {
+| Some(root) => ReactDOM.render(<App />, root)
```

Comment:
Do you know why `root` is underlined?

<img width="713" alt="image" src="https://github.com/melange-re/melange-for-react-devs/assets/220424/0cf47f50-df55-4521-b2da-9a560f67ed14">

Reply:
No idea. Do you think I should try to fix it?

Reply:
I don't know. Did you get the grammar from the vscode-ocaml extension? I can't see that behavior in vscode itself.

Reply:
> Did you get the grammar from the vscode-ocaml extension?

https://github.com/melange-re/melange-for-react-devs/blob/1a4b87d0f374dcf21525b8973d64143fae0bcf72/docs/.vitepress/config.js#L5-L6

For reference, it looks like anything that appears as the lighter red color in VS Code (in default light theme) will be underlined in vitepress syntax highlighting.

Reply:
I'll try playing around with shiki config to see if there's a theme that doesn't underline these parts. Do you think it's a blocker?

Reply:
Sounds good. The underlying is a bit distracting, but it might be just me.